packages/api/src/utils/client/httpClient.ts

```diff
@@ -196,7 +196,7 @@ export class HttpClient implements IHttpClient {
               this.logger?.debug("Requesting fallback URL", {routeId, baseUrl: printableUrl, score: this.urlsScore[i]});
             }
 
-            const i_ = i; // Keep local copy of i variable to index urlScore after requestWithBody() resolves
+            const localIndex = i; // Keep local copy of i variable to index urlScore after requestWithBody() resolves
```

Comment:
can't we just add a disable here? having the code reformatted into multiple lines is not great for readability here